### PR TITLE
Update some ui

### PR DESCRIPTION
**Pull request by banguyenvareal**

### CODE REVIEW

File: app/views/book_requests/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, 'Update Book Request')  %>
+<h1>Update Your Book Request </h1>
```
Comment (mrcuong84) [inline app/views/book_requests/edit.html.erb]: i18n

File: app/views/book_requests/new.html.erb [hunk 2]
```diff
@@ -1,49 +1,3 @@
 <% provide(:title, 'Create Book Request')  %>
 <h1>Create Book Request </h1>
```
Comment (mrcuong84) [inline app/views/book_requests/new.html.erb]: i18n

**Review by mrcuong84 — APPROVED** [review]

**Review by dinhvareal — APPROVED** [review]

File: app/views/users/book_requests/_list.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+          <p class="book-request-detail"><%= t('book_request.quantity') %>
+            <%= book_request.quantity %>
+          </p>
+          <p class="book-request-detail"><%= t('book_request.budget') %>
+            <%= book_request.budget %>
+          </p>
+          <p class="book-request-detail"><span title="<%= book_request.comment %>">
+            <%= truncate(book_request.comment, length: 25) %></span>
+          </p>
+          <a href="<%= edit_user_book_request_path current_user, book_request %>"><span class="glyphicon glyphicon-edit"></span></a>
```
Comment (dinhvareal) [inline app/views/users/book_requests/_list.html.erb]: use link_to

File: app/views/users/book_requests/_list.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<% book_requests&.each do |book_request| %>
```
Comment (hieunguyentrung) [inline app/views/users/book_requests/_list.html.erb]: Limit use `&`. Remove it. !

File: app/controllers/users/book_requests_controller.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+module Users
+  class BookRequestsController < ApplicationController
+    def index
+      @book_requests = User.find(params[:user_id])
```
Comment (hieunguyentrung) [inline app/controllers/users/book_requests_controller.rb]: Breake into `book_user` method to find user.

File: app/views/books/_book_list.html.erb [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<% books.each do |book| %>
+<% @books.each do |book| %>
```
Comment (hieunguyentrung) [inline app/views/books/_book_list.html.erb]: Don't use instance variable inside the partial view.

Comment (hieunguyentrung) [reply app/views/books/_book_list.html.erb]: `_list.hmtl.erb` is enough.

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -10,6 +10,7 @@
       <div class="col-sm-2"></div>
       <div class="col-sm-7">
         <ul class="nav navbar-nav navbar-right">
+          <li><%= link_to "Book Requests", book_requests_path %></li>
```
Comment (hieunguyentrung) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/users/show.html.erb [hunk 8]
```diff
@@ -1,3 +1,11 @@
+<div class="row">
+  <nav class="navbar navbar-default col-sm-12 tab-option">
+      <ul class="nav navbar-nav">
+        <li><%= link_to "Books", "#" %></li>
+        <li><%= link_to "Book Requests", user_book_requests_path(current_user)%></li>
```
Comment (hieunguyentrung) [inline app/views/users/show.html.erb]: i18n.

**Review by hieunguyentrung — CHANGES_REQUESTED** [review]

**Review by hieunguyentrung — APPROVED** [review]

**Pull request merged**